app/src/tests/components/contents/downloads/Downloads.test.tsx

```diff
@@ -24,7 +24,7 @@ describe("download component", () => {
       <SWRConfig value={{ dedupingInterval: 0 }}>
         <MemoryRouter initialEntries={[`/${packet.name}/${packet.id}/downloads`]}>
           <Routes>
-            <Route element={<PacketLayout />} path="/:packetName/:packetId">
+            <Route element={<PacketOutlet packetId={packet.id} />}>
```

Comment:
Why this change?

Reply:
because packetlayout wraps packetoutlet. packetlayout needs more dependencies now (user) as well thus if we are testing `Downloads.test.tsx` don't need to test the layout as well, outlet should suffice